recipes/libfabric/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "1.22.0" %}
+{% set build = 0 %}
+
+package:
+  name: libfabric
+  version: {{ version }}
+
+source:
+  url: https://github.com/ofiwg/libfabric/releases/download/v{{ version }}/libfabric-{{ version }}.tar.bz2
+  sha256: 485e6cafa66c9e4f6aa688d2c9526e274c47fda3a783cf1dd8f7c69a07e2d5fe
+
+build:
+  number: {{ build }}
+
+  run_exports:
+    - {{ pin_subpackage('libfabric', max_pin='x.y') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - libtool  # [unix]
+    - make  # [unix]
+  host:
+    - libnl  # [linux]
+    - numactl  # [linux]
+  run:
+
+test:
+  commands:
+    - fi_info --version  # [osx]
+    - fi_info --list  # [linux]
```

Comment:
is `fi_info --list` failing on osx? If not, this can be `# [not win]`

Reply:
OK

Reply:
```suggestion
    - fi_info --list
```